### ISSUE TITLE
Reproducibility of metrics

### ISSUE THREAD

**Issue by Caitianshun** [1231234zhan/mmlphuman#17]
Hello author, I used your README to rebuild the virtual environment of this project on multiple machines like 3090, 5090, and A100, but I ran into the same issue of slowed training speed. Towards the later stages, it drops to 3 sec/iter, making it unrealistic to successfully complete 800k iterations. Later, I tried the LPIPS solution mentioned in issue 6. Although the speed recovered, the metrics for the Subject00 and AvatarRex ZZr datasets both dropped compared to the paper. For Subject00, the metrics are even lower than Animatable Gaussians. I trained entirely according to the dataset YAML configuration files in your paper's appendix and under config folder. Could you tell me what the likely reason for this is?

**Comment by 1231234zhan** [comment]
Can I know what metric number you get after training? If the number is a little bit lower than AnimatableGaussian and the visual results don't have problems and look the same, it could be treated as a normal phenomenon. If it's much lower than AnimatableGaussian or the visual results looks weird, maybe you could check if there is some problems with the data or the training process. Make sure to use PCA pose projection when doing novel pose animation, because it can greatly affect the results.

**Comment by Caitianshun** [comment]
<img width="631" height="213" alt="Image" src="https://github.com/user-attachments/assets/25ae9a0a-02d8-4c8e-87eb-0646245b7591" />
<img width="294" height="224" alt="Image" src="https://github.com/user-attachments/assets/4a9a34f7-41e4-4bf9-9dbd-9498ecad04d6" />
<img width="274" height="222" alt="Image" src="https://github.com/user-attachments/assets/52d88ef8-b82b-4e6d-b91c-d1dae696323c" />
<img width="855" height="615" alt="Image" src="https://github.com/user-attachments/assets/47dd6479-0e08-49fe-b750-164efe68ab86" />  Hello, author. Sorry for replying so late and missing your message. Thanks a lot for your answer. The few images in the attachment are my modified LPIPS loss function, the three metrics for the AVR and subject00 datasets, and one image showing the metrics for the two datasets from your paper. In the new pose testing, I used PCA projection, and the rendered images don’t have major issues. Could you please take a look and see if the training metrics I mentioned above are normal? During training, I only modified the loss function and also used the optional template mesh.

**Comment by Caitianshun** [comment]
I noticed that in the metrics I trained, for the trained poses, the LPIPS is lower than animatable Gaussians, and for the novel poses, the metrics are all lower than animatable Gaussians. Since we plan to use the metrics from your paper as a baseline later, I wanted to ask if you have any ideas on how to solve these issues, or are these metrics completely normal?

**Comment by 1231234zhan** [comment]
We use LPIPS with `net='vgg'`, which is consistent with AnimatableGaussian. Compared with `net='alex'`, it is slower but usually produces better results. Similarly, we also use the VGG model for evaluation.

Results may vary from person to person. Some specific factors, such as how image masks are handled and whether the images are properly aligned, can significantly affect the quantitative results. I cannot look into the issue in detail, based on my observations, under a fair setting, if the metrics of mmlphuman are not significantly lower than AnimatableGaussian and the visual quality of the two methods is similar, then the results are normal. This usually also means that you need to train all the baselines yourself, rather than directly using the numbers reported in the paper tables.

**Comment by Caitianshun** [comment]
Okay, thank you very much for your reply. I also tested the VGG version with the revised loss function that you mentioned in other issues before, but the metrics are almost the same as I mentioned above, and they still end up being lower than animatableGaussian. I got it, I will compare it with my own trained baseline .
